test/github-url.ts

```diff
@@ -3,6 +3,8 @@ import test from 'ava';
 import './fixtures/globals';
 import GitHubURL from '../source/github-helpers/github-url';
 
+document.head.insertAdjacentHTML('beforeend', '<a hidden="" data-hotkey="t" data-pjax="true" href="https://github.com/avajs/ava/find/master"></a>');
```

Comment:
```suggestion
document.head.insertAdjacentHTML('beforeend', '<a hidden="" data-hotkey="t" data-pjax="true" href="https://github.com/sindresorhus/refined-github/find/Slash%2Fslash"></a>');
```

Reply:
Actually these can't be here. They need to be inside the `test()` function that uses it